concepticondata/concepticon.tsv

```diff
@@ -3906,3 +3906,5 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3920	GENIPA	Agriculture and vegetation	Either a species of trees in the family Rubiaceae native to the tropical forests of Americas or its fruit (also referred to as “genipap”). A Genipa tree typically has large smooth or hairy leaves and ovoid-shaped fruit with large and flat seeds.	Person/Thing	
 3921	TINAMOU	Animals	Any of the 47 sedentary, ground-dwelling bird species of the order Tinamiformes found in Central and South America. It bears a strong resemblance to patridges but has limited flight capability.	Person/Thing	
 3922	PACU	Animals	A freshwater species of fish belonging to the family Serrasalmidae and related to piranha.	Person/Thing	
+3923	FEMALE ANIMAL	Animals	A female animal (Noun)	Property	
```

Comment:
The ontological category would be "Person/Thing" for both.

Reply:
riiight, of course :) fixed.